### DISCUSSION TITLE
Sent CHIA tokens did not arrive in my Chia Native Wallet

### DISCUSSION THREAD

**Discussion by lpcms** [Chia-Network/chia-blockchain#18025] [Support]
Hello!
Sent CHIA tokens did not arrive in my Chia Native Wallet.
Wallet version: 2.3.0 for Widnows
Wallet address: xch1t864ykwzdapvp65uepf44sxvdzfzur5df7rqw3scstdzuagsckas7fresg
Transaction hash: https://xchscan.com/en/txns/0x61b7dddd41fa3989a43566b3e1b94ead68f82d876fe930de25b9f7dd6a7380d6

But I don’t see tokens and transactions on my Chia Native Wallet?
Please tell me what could be the problem?
<img width="1002" alt="sshot-1" src="https://github.com/Chia-Network/chia-blockchain/assets/22012710/b6bf4784-d193-429f-8f26-1bed8b1c1119">

**Comment by BrandtH22** [comment]
Hey @lpcms , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia). 

As for the funds, take note of the wallet sync icon in the top right of the screenshot and that it is yellow/orange. This means that the wallet is still syncing. 

If you click on that icon it will show you your peers and how much data you are downloading from them. 

If you are not connected to any peers or are not syncing from those peers then it would be worthwhile to look at your debug log which can be found at `~/.chia/mainnet/log/debug.log` for any errors or warnings. (note `~` in windows means the folder with your username and `.chia` is a hidden folder so in finder you need to make sure the view options are showing hidden folders).

The logs generally give a good indication as to what the issues are with syncing but if you have any questions don't hesitate to jump into our discord server or respond here.

**Reply by lpcms** [reply]
Hello!
Thank you very much for your help!
The transaction has now appeared in the wallet.
Now I'm calm, thank you!

**Comment by BrandtH22** [comment]
That is great to hear! Thank you for the confirmation and I am glad the funds are visible! 
I am going to close this ticket but do not hesitate to open a new one or join our discord server if other questions arise.

https://discord.gg/chia
